.travis.yml

```diff
@@ -0,0 +1,39 @@
+# Travis configuration file using the build matrix feature
+# Read more under http://docs.travis-ci.com/user/build-configuration/
+# THIS SCRIPT IS SUPPOSED TO BE AN EXAMPLE. MODIFY IT ACCORDING TO YOUR NEEDS!
+
+sudo: false
+language: python
+virtualenv:
+  system_site_packages: false
+matrix:
+  fast_finish: true
+  include:
+    - python: 3.6
```

Comment:
As with spin1_beam_model, I think it'd be good to build for py3.7 too, but that can be done in a separate PR when we've got full-fledged tests going.